scilpy/tracking/tools.py

```diff
@@ -127,6 +129,53 @@ def resample_streamlines(streamlines, num_points=0, arc_length=False):
     return resampled_streamlines
 
 
+def smooth_line_gaussian(streamline, sigma):
+    if sigma < 0.00001:
+        ValueError('Cant have a 0 sigma with gaussian.')
+
+    nb_points = int(length(streamline))
+    if nb_points < 2:
+        logging.debug('Streamline shorter than 1mm, corner cases possible.')
```

Comment:
Does this presume that the streamlines are resampled to voxmm ?

Reply:
Yesm either voxmm or rasmm